src/ui/tabpopup.c

```diff
@@ -403,6 +403,7 @@ meta_ui_tab_popup_new (const MetaTabEntry *entries,
   gtk_container_set_border_width (GTK_CONTAINER (grid), 16);
   gtk_container_add (GTK_CONTAINER (popup->window), frame);
   gtk_container_add (GTK_CONTAINER (frame), vbox);
+  gtk_style_context_add_class (gtk_widget_get_style_context (frame), "mate-tabwin");
```

Comment:
nitpick: why not call the class `tabpopup` or `tab-popup` (or even `tab-popup-frame`)? It would match the rest of the code (which refers to this window as such).

Reply:
Sure, but:
```
~/dev $ git -C xfce/xfwm4 grep tabwin | wc -l
351
~/dev $ git -C mate/marco grep tabwin | wc -l
0
```
xfwm uses the `tabwin` name heavily in their window manager, most of the code in `tabwin.c`. In contrast, marco has `tabpopup.c` and uses the name `tabpopup` throughout.